nekoyume/Assets/_Scripts/UI/Tutorial/GuideArrow.cs

```diff
@@ -131,5 +132,11 @@ public void PlaySfx()
         {
             AudioController.instance.PlaySfx(AudioController.SfxCode.GuideArrow);
         }
+
+        public override void Skip(System.Action callback)
+        {
+            _arrow.Play(_arrow.GetCurrentAnimatorStateInfo(0).shortNameHash, -1, 1);
```

Comment:
Why do you use `0` to get status, but `-1` for play?
```suggestion
            _arrow.Play(_arrow.GetCurrentAnimatorStateInfo(0).shortNameHash, 0, 1);
```

Reply:
It's my mistake. I'll fix it!